test/functional/example_test.py

```diff
@@ -70,7 +70,7 @@ def custom_function():
 class ExampleTest(BitcoinTestFramework):
     # Each functional test is a subclass of the BitcoinTestFramework class.
 
-    # Override the set_test_params(), add_options(), setup_chain(), setup_network()
+    # Override the set_test_params(), add_arguments(), setup_chain(), setup_network()
```

Comment:
All of our arguments are options, so our name of this method can be left untouched.